CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changelog
 ## 1.4.0 (04-28-2017)
-* [Breaking change] [getThreadHistory](/DOCS.md#getThreadHistory): update parameters
-* [New] [setMessageReaction](/DOCS.md#setMessageReaction)
+* [Breaking change] [getThreadHistory](/DOCS.md#getThreadHistory): update parameters - no more start & end params; replaced with amount
```

Comment:
I know that it is not new, but it would make it clear to other users if they haven't used timestamp before (only start and end) 
@Schmavery what do you think?